src/services/authentication/hooks/index.js

```diff
@@ -78,6 +78,7 @@ const injectUserId = async (context) => {
 			if (account) {
 				context.params.payload = {};
 				context.params.payload.accountId = account._id;
+				context.params.payload.forcePasswordChange = account.forcePasswordChange;
```

Comment:
account.forcePasswordChange ? You have no information over forcePasswordChange in account. 
Do you want account.userId? 
At the moment we have at this point no populated userId. 
To re-request the user over the userId is a high negative performance impact. 

app.service('/accounts').patch(accountId, {
		password: newPassword,
		userForcedToChangePassword: true,
}, params); 

Do you want this payload.forcePasswordChanges?

Reply:
Do you use payload.forcePasswordChanges in client?

Reply:
I have removed this line. It is no longer needed

Reply:
the line is still there though..

Reply:
:+1:  double done